WEB/Src/DependencyCollector/DependencyCollector/DependencyCollector.csproj

```diff
@@ -60,7 +60,7 @@
     <PackageReference Include="Microsoft.Extensions.DiagnosticAdapter" Version="1.1.0" />
     <PackageReference Include="Microsoft.Extensions.PlatformAbstractions" Version="1.1.0" />
     <PackageReference Include="System.Diagnostics.StackTrace" Version="4.3.0" />
-    <PackageReference Include="System.Data.SqlClient" Version="4.3.1" />
+    <PackageReference Include="System.Data.SqlClient" Version="4.7.0" />
```

Comment:
Can we simply get rid of this dependency? Dep collector shouldn't need this as it relies on reflection on DiagnosticSource callbacks. (the whole beauty of using DiagSource is to avoid package version issues i think..)

Reply:
Done.